base/system/services/database.c

```diff
@@ -1364,23 +1468,24 @@ ScmControlService(HANDLE hControlPipe,
                         &Overlapped);
     if (bResult == FALSE)
     {
-        DPRINT("WriteFile() returned FALSE\n");
+        DPRINT("(%S, %d) WriteFile() returned FALSE\n", lpService->lpServiceName, dwControl);
```

Comment:
Not sure we need all these %s, %d in these dprints.

Reply:
Left for now. Since they are DPRINT's there will be no debug spam. These may be helpful for future trouble-shooting.